Doc/howto/isolating-extensions.rst

```diff
@@ -461,7 +461,7 @@ Module State Access from Slot Methods, Getters and Setters
 
    .. After adding to limited API:
```

Comment:
This part doesn't render ([see page](https://docs.python.org/dev/howto/isolating-extensions.html#module-state-access-from-slot-methods-getters-and-setters)) so unsure what this should look like.

Reply:
It's a reST "comment", which doesn't render; not sure if its intended to be or not, since it does use reST syntax within itself.

@encukou any insight here?